creation/web_base/singularity_lib.sh

```diff
@@ -2106,6 +2113,10 @@ setup_classad_variables() {
     export GWMS_SINGULARITY_STATUS=$(get_prop_str ${_CONDOR_MACHINE_AD} GWMS_SINGULARITY_STATUS)
     export GWMS_SINGULARITY_PATH=$(get_prop_str ${_CONDOR_MACHINE_AD} SINGULARITY_PATH)
     export GWMS_SINGULARITY_VERSION=$(get_prop_str ${_CONDOR_MACHINE_AD} SINGULARITY_VERSION)
+    export GWMS_CONTAINERSW_STATUS=$(get_prop_str ${_CONDOR_MACHINE_AD} GWMS_CONTAINERSW_STATUS)
```

Comment:
I do not see any reference to the variable `GWMS_CONTAINERSW_STATUS`, used in the last part of this line, in any of the modified files. Just making sure that this does not cause any issues when in execution. 

I do see a variable named `GWMS_CONTAINERSW_MODE` but is not used anywhere in the code.

Reply:
These variables are there to try to future proof. Currently there is only one container sw (singularity in its variants).

I removed GWMS_CONTAINERSW_STATUS. Thinking better it does not make sense yet since the container sw selection may use different mechanisms. Maybe in the future when things are more defined.

GWMS_CONTAINERSW_MODE/CONTAINERSW_MODE are staying OSG and others outside GWMS may start using it.

I pushed an update